protected/commands/InvalidationQueryCommand.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class InvalidationQueryCommand extends CConsoleCommand
```

Comment:
**thought:** This Command class is only used by the `InvalidationQueryCommandCest` functional test. Perhaps it would be better to move the two functions into the `tests/_support/Helper/Functional` class - I think they should be callable in your `InvalidationQueryCommandCest` from there.

Reply:
I tried to create this function in `tests/_support/Helper/Functional.php`:
```
public function getLatestCreateUsingQueryFromMainConfigFile(string $dataset_id)
    {
        $invalidationQuery = preg_replace("/@id/", $dataset_id, Yii::app()->params['invalidationQuery']);
        $rows = Yii::app()->db->createCommand($invalidationQuery)->queryRow();
        foreach ($rows as $row) {
            print_r($row);
        }
    }
  ```
  Then I updated the test to:
  ```
  public function tryToGetTheLatestCreatedAtFromTableDatasetlog(FunctionalTester $I)
    {
        $date = date('Y-m-d H:i:s');
        $dataset_id = 8;
        $I->haveInDatabase("dataset_log", [
                "dataset_id" => $dataset_id,
                "message" => "File Pygoscelis_adeliae.RepeatMasker.out.gz updated",
                "created_at" => $date,
                "model" => "File",
                "model_id" => 17679,
                "url" => "/adminFile/update/id/17679"
            ]);

//        $output = shell_exec(" ./protected/yiic invalidationquery getlatestcreateusingqueryfrommainconfigfile");
        $output = $I->getLatestCreateUsingQueryFromMainConfigFile($dataset_id);
        codecept_debug($output);
        $I->assertContains($date, $output);
    }
 ```
When I ` % docker-compose run --rm test ./vendor/codeception/codeception/codecept run functional InvalidationQueryCommandCest:tryToGetTheLatestCreatedAtFromTableDatasetlog --debug -vvv`, would get this error:
```
1) InvalidationQueryCommandCest: Try to get the latest created at from table datasetlog
 Test  tests/functional/InvalidationQueryCommandCest.php:tryToGetTheLatestCreatedAtFromTableDatasetlog
                                        
  [Error] Class 'Helper\Yii' not found  
```